src/HealthChecks.Elasticsearch/HealthChecks.Elasticsearch.csproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Elastic.Clients.Elasticsearch" Version="8.14.3" />
-    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="8.0.6" />
```

Comment:
Is there a specific reason this was at `8.0.6` and not `8.0.0` like everything else?

Reply:
No reason.

This package updated in https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/commit/42c2c732a5369849d59f9d79cdd4be497da98786#diff-c1972dbf9f5f7338faa0c70f37d85bd82ca888d0141690c7204647f768e47c0c

And I think we can go back to `8.0.0` because we did not release the Elasticsearch package with that change.

Reply:
OK this sounds good to me. Looks like it was a mistake to update it to `8.0.6` in the first place.